test/fixtures/render/consul/override.json

```diff
@@ -0,0 +1,26 @@
+{
+  "cfg": {
+    "website": false,
+    "server": {
+      "datacenter": "IN_OVERRIDE_JSON"
+    },
+    "ports": {
+      "dns": 6666,
+      "http": 6667,
+      "https": 6668
+    }
+  },
+  "sys": {
+    "ip": "9.9.9.9"
+  }, 
+  "svc": {
+    "members": [
+      { "alive": true, "sys": { "ip": "1.1.1.1" }},
+      { "alive": true, "sys": { "ip": "2.2.2.2" }},
+      { "alive": true, "sys": { "ip": "3.3.3.3" }}
+    ]
+  },
+  "pkg": {
+    "svc_config_path": "/home/foo/bar"
+  }
+}
```

Comment:
Not for this PR, but it would be interesting if we could somehow incorporate in our existing [JSON schema definition for templating data](https://github.com/habitat-sh/habitat/blob/master/components/sup/doc/render_context_schema.json) into this rendering tool. 

I imagine it would be quite useful to know if you've supplied testing data that doesn't actually conform to the data that your templates would actually encounter in the real world.

Reply:
:+1: dig it.   #6206